sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -199,5 +207,29 @@ public virtual async Task<Response<AccountProperties>> GetAccountPropertiesAsync
         }
 
         #endregion
+
+        /// <summary>
+        /// Used as part of argument validation. Attempts to create a <see cref="Guid"/> from a <c>string</c> and
+        /// throws an <see cref="ArgumentException"/> in case of failure.
+        /// </summary>
+        /// <param name="modelId">The model identifier to be parsed into a <see cref="Guid"/>.</param>
+        /// <param name="paramName">The original parameter name of the <paramref name="modelId"/>. Used to create exceptions in case of failure.</param>
+        /// <returns>The <see cref="Guid"/> instance created from the <paramref name="modelId"/>.</returns>
+        /// <exception cref="ArgumentException">Happens when parsing fails.</exception>
+        private static Guid ValidateModelId(string modelId, string paramName)
```

Comment:
Would it make sense to have this in a common place so we don't duplicate code?

Reply:
Discussed offline: the method has been moved to a new `ClientCommon` class.